### PR TITLE
fix(scripts-generators): make create-component work properly on windows and improve DX

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

`yarn create-component`

execution introduces sets package in invalid state:
- lint errors
- doesn't set export map correctly when executing on windows

## New Behavior

`yarn create-component`
- empty `export {}` is being removed on 1st `create-component` execution
- `export * from` is expanded on every `create-component` run
- `migrate-converged-pkg` properly setups export maps when invoked on windows OS
- eslint-plugin `@fluentui/max-length` rule was turned off for v9 packages (it doesn't make much sense as we have prettier that will enforce line length automatically without any churn to the user (DX)

## Related Issue(s)

* Fixes https://github.com/microsoft/fluentui/pull/26529/commits/08c84ee99739e969e98e352d02a08fb0c8528c53

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                      |     Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                 Change |
| :----------------------------------------------------------------------------------------------------- | ---------------------------: | ----------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr> | `118.656 kB`<br />`36.11 kB` | `118.683 kB`<br />`36.127 kB` | `27 B` <br />`17 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                                                                    |          `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>                                                    |          `554 B`<br />`348 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>                                                      |    `3.182 kB`<br />`1.308 kB` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `80.097 kB`<br />`24.712 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |   `86.462 kB`<br />`21.76 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `53.054 kB`<br />`14.661 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `15.42 kB`<br />`6.167 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |   `69.272 kB`<br />`18.96 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |    `23.78 kB`<br />`7.542 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |    `24.67 kB`<br />`7.825 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |    `25.83 kB`<br />`7.451 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `33.684 kB`<br />`8.956 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `40.704 kB`<br />`10.408 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `38.364 kB`<br />`10.268 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `45.934 kB`<br />`11.697 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `49.647 kB`<br />`10.967 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `78.082 kB`<br />`22.521 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |   `72.92 kB`<br />`21.108 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `9.04 kB`<br />`3.807 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `10.916 kB`<br />`4.487 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `9.847 kB`<br />`4.159 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                              |  `32.768 kB`<br />`10.213 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/CheckboxField.fixture.js'>CheckboxField</abbr>                                                                    |  `39.673 kB`<br />`12.094 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `82.678 kB`<br />`26.408 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/ComboboxField.fixture.js'>ComboboxField</abbr>                                                                    |  `79.095 kB`<br />`26.183 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `81.346 kB`<br />`26.144 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `60.588 kB`<br />`16.969 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `188.355 kB`<br />`52.98 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.751 kB`<br />`11.453 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `83.576 kB`<br />`24.951 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `17.042 kB`<br />`6.153 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                                       |   `17.605 kB`<br />`6.518 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `11.259 kB`<br />`4.473 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `24.848 kB`<br />`8.136 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/InputField.fixture.js'>InputField</abbr>                                                                             |   `34.99 kB`<br />`11.072 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `9.817 kB`<br />`4.067 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `12.306 kB`<br />`5.073 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `118.221 kB`<br />`36.568 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             |  `121.29 kB`<br />`37.071 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `11.139 kB`<br />`4.233 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |   `59.892 kB`<br />`16.55 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `105.773 kB`<br />`32.613 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `11.122 kB`<br />`4.102 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |   `20.856 kB`<br />`7.766 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |   `13.573 kB`<br />`5.232 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressField.fixture.js'>ProgressField</abbr>                                                                    |   `23.805 kB`<br />`8.339 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `16.818 kB`<br />`6.247 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `32.353 kB`<br />`10.504 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |    `14.822 kB`<br />`5.91 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroupField.fixture.js'>RadioGroupField</abbr>                                                                   |   `24.895 kB`<br />`9.045 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |    `24.14 kB`<br />`8.271 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/SelectField.fixture.js'>SelectField</abbr>                                                                          |  `33.316 kB`<br />`10.755 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `32.768 kB`<br />`10.438 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/SliderField.fixture.js'>SliderField</abbr>                                                                          |  `42.599 kB`<br />`13.345 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `45.161 kB`<br />`13.035 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButtonField.fixture.js'>SpinButtonField</abbr>                                                              |  `54.361 kB`<br />`15.594 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `20.552 kB`<br />`6.658 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `27.902 kB`<br />`8.647 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/SwitchField.fixture.js'>SwitchField</abbr>                                                                          |  `34.687 kB`<br />`10.541 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `129.423 kB`<br />`35.243 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `42.971 kB`<br />`11.781 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `127.672 kB`<br />`32.893 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `81.106 kB`<br />`20.161 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `80.535 kB`<br />`20.005 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `12.261 kB`<br />`4.819 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `15.571 kB`<br />`5.23 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `26.331 kB`<br />`8.581 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/TextareaField.fixture.js'>TextareaField</abbr>                                                                    |  `36.469 kB`<br />`11.483 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `30.908 kB`<br />`6.714 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |   `17.776 kB`<br />`5.141 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |   `43.634 kB`<br />`15.26 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e1de0e36aff891fd57b3f93447844dcdbdf9c45f'>e1de0e36aff891fd57b3f93447844dcdbdf9c45f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Avatar-mount_base.html">1275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Avatar-mount.html">1280</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Button-mount_base.html">926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Button-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProvider-mount_base.html">1520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProvider-mount.html">1539</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-mount_base.html">578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-mount.html">576</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">548</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">581</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/InfoButton-mount_base.html">520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/InfoButton-mount.html">519</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/MakeStyles-mount_base.html">1959</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/MakeStyles-mount.html">1962</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Persona-mount_base.html">2822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/Persona-mount.html">2756</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/SpinButton-mount_base.html">2367</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26569/perf-test-react-components/SpinButton-mount.html">2368</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-eslint-plugin-9c309582-f6de-425d-bc96-c927db2786da.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-eslint-plugin-9c309582-f6de-425d-bc96-c927db2786da.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26569&commitId=41b053edc30efd78d4e330c923d0fc971cd0b71c&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: packages/eslint-plugin/src/configs/base.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ module.exports = {
      * @see https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin
      */
     ...getNamingConventionRule(),
+    '@fluentui/max-len': 'off',
```
Comment (Hotell) [inline packages/eslint-plugin/src/configs/base.js:16]: this rule doesn't make much, especially with severity "error", 

it was also triggered for new generated component by `create-component`

in general we have prettier to enforce line length so turning this off for new codebase

File: scripts/generators/create-component/index.ts [hunk 3]
```diff
@@ -117,6 +117,11 @@ module.exports = (plop: NodePlopAPI) => {
             stdio: 'inherit',
           });
 
+          execSync(`yarn workspace ${data.packageNpmName} lint --fix`, {
```
Comment (Hotell) [inline scripts/generators/create-component/index.ts:120]: will unwrap `export *`

File: scripts/generators/create-component/index.ts [hunk 4]
```diff
@@ -139,18 +144,32 @@ const checkIfComponentAlreadyExists = (data: Data): string => {
 const appendToPackageIndex = (data: Data): string => {
   const { componentName, packageName, packagePath } = data;
 
-  // get the package index file path
   const indexPath = path.join(packagePath, 'src/index.ts');
+  const indexContent = cleanCreatePackageTemplate(indexPath);
```
Comment (Hotell) [inline scripts/generators/create-component/index.ts:148]: removes empty `export {}` on first create-component  run

File: tools/generators/migrate-converged-pkg/index.ts [hunk 5]
```diff
@@ -669,7 +669,7 @@ function updatePackageJson(tree: Tree, options: NormalizedSchemaWithTsConfigs) {
     return json;
 
     function normalizePackageEntryPointPaths(entryPath: string) {
-      return './' + path.normalize(entryPath);
+      return './' + path.posix.normalize(entryPath);
```
Comment (Hotell) [inline tools/generators/migrate-converged-pkg/index.ts:672]: this will produce `\\` on windows, we always wanna use posix

**Review by ling1726 — APPROVED** [review]

**Pull request merged**